pkg/sbom/generator/spdx/spdx.go

```diff
@@ -197,7 +197,7 @@ func (sx *SPDX) imagePackage(opts *options.Options) (p *Package) {
 }
 
 // apkPackage returns a SPDX package describing an apk
-func (sx *SPDX) apkPackage(opts *options.Options, pkg *repository.Package) (p Package, err error) {
```

Comment:
Nope, this isn't to satisfy an interface. We can just remove the err return. I'll send a PR if you don't.

Reply:
https://github.com/chainguard-dev/apko/pull/371